### DISCUSSION TITLE
fluentd unknown plugin elasticsearch

### DISCUSSION THREAD

**Discussion by vasanthaneppali** [fluent/fluentd#3744] [General]
### Describe the bug

Hi
i am new to new fluentd, i am trying to upgrade the fluentd version from v1.11-debian to v1.14-debian.
i am running fluend with docker container
while copying volume i am unable to start the fluentd, in the logs it is showing like
2022-05-07 01:46:57 +0530 [error]: config error file="/fluentd/etc/fluent.conf" error_class=Fluent::ConfigError error="Unknown output plugin 'elasticsearch'. Run 'gem search -rd fluent-plugin' to find plugins"
please help

### To Reproduce

....

### Expected behavior

Want the fluentd container to be up

### Your Environment

```markdown
- Fluentd version:
- TD Agent version:
- Operating system:
- Kernel version:
```

### Your Configuration

```apache
<match *.**>
  @type copy
  <store>
    @type elasticsearch
    host elasticsearchoss
    port 9200
    logstash_format true
    logstash_prefix fluentd
    logstash_dateformat %Y%m%d
    include_tag_key true
    type_name access_log
    tag_key @log_name
    flush_interval 1s
    rollover_index true # defaults to false
    index_date_pattern "now/w{xxxx.ww}" # defaults to "now/d"
```

### Your Error Log

```shell
2022-05-07 14:33:57 +0530 [info]: parsing config file is succeeded path="/fluentd/etc/fluent.conf"
2022-05-07 14:33:58 +0530 [info]: gem 'fluentd' version '1.14.0'
2022-05-07 14:33:58 +0530 [error]: config error file="/fluentd/etc/fluent.conf" error_class=Fluent::ConfigError error="Unknown output plugin 'elasticsearch'. Run 'gem search -rd fluent-plugin' to find plugins"
```

### Additional context

I am running it as docker image

**Comment by ashie** [comment]
It's not a bug nor a feature request, so I moved this to discussion.

**Comment by fujimotos** [comment]
>  i am trying to upgrade the fluentd version from v1.11-debian to v1.14-debian.
> i am running fluend with docker container
> while copying volume i am unable to start the fluentd, in the logs it is showing like

My psychic guess is that your Ruby version was changed
during the upgrade.

Since gems are stored per Ruby version, your new Fluentd
is unable to locate `elasticsearch`.

Check your current Gem path, and make sure that
`fluent-plugin-elasticsearch` is available there.
